_includes/footer.md

```diff
@@ -4,10 +4,7 @@
 [![Apache Software Foundation][ASF_IMG]{: #asf-logo height="100"}][ASF]
 
 Copyright &copy; 2011-{{ site.time | date: '%Y' }} The Apache Software
-Foundation, Licensed under the [Apache License, Version 2.0][AL2]. Apache
```

Comment:
I added the trademark to the main page using a front matter variable in 5d113467a12.  Does this work?